src/nti/site/subscribers.py

```diff
@@ -68,11 +68,12 @@ def getSiteManager(self):
         return self.__site_manager
 
     @non_overridable
-    def setSiteManager(self, new_man):
+    def setSiteManager(self, new_man): # pylint:disable=unused-argument
         raise ValueError("Cannot set site manager on proxy")
 
+
 @component.adapter(ISite, IBeforeTraverseEvent)
-def threadSiteSubscriber(new_site, event):
+def threadSiteSubscriber(new_site, _event):
```

Comment:
The changes in this function might be easier to see in split (as opposed to unified) diff mode.